### ISSUE TITLE
VirtualRadarServer - not showing routes, silhouettes or flags

### ISSUE THREAD

**Issue by SingingDwarf** [TheBiggerGuy/docker-ads-b#8]
Virtual Radar Server is not displaying aircraft silhouettes, flags or routes - despite the aircraft transmitting valid ICAO and callsign.

**Comment by TheBiggerGuy** [comment]
Could you please confirm you have configured VRS to read from the dump1090 source? Unfortunately I have yet to make it auto configure to the correct source.

Settings can be found via: http://localhost:8080/VirtualRadar/WebAdmin/Settings.html

![Screenshot from 2019-05-01 22-47-52](https://user-images.githubusercontent.com/152316/57044693-482d4800-6c63-11e9-864f-a668a288adea.png)

![Screenshot from 2019-05-01 22-49-12](https://user-images.githubusercontent.com/152316/57044717-6135f900-6c63-11e9-98f1-9560d905cb11.png)

**Comment by SingingDwarf** [comment]
Yes, I have it configured correctly to the MODE-S transmission from dump1090 (not MLAT, as that is not working correctly subject to a different issue).

I see the same as you in the example posted above - under the aircraft list, i.e. no silhouette, flag or route, although the reg, ICAO, callsign, alt and speed are all displayed correctly.

I'm guessing its because the flags and silhouettes are not distributed with the build. From what I can gather on the VRS site, they need to be [downloaded and stored somewhere locally](https://forum.virtualradarserver.co.uk/viewtopic.php?t=929).

As to the route, this appears to come from a [database file](http://www.virtualradarserver.co.uk/flightroutes.aspx#database-files), which I assume is also missing from the distribution. 

Presumably both the silhouettes/flags and database could be stored outside of the docker image and mapped from it, with the path(s) then needing to be updated via the web admin plugin. Need to brush up on my docker skills and see if this is possible.

Presumably a similar issue is causing the VRS reports not to run - the data from the days flights has no database to be written to?

**Comment by TheBiggerGuy** [comment]
Could you please pull the latest build (eg. ```docker-compose pull```). If it still is not working I will have to look deeper into VRS. The above screenshot was taken from the latest docker image so I cannot currently reproduce the issue.

**Comment by TheBiggerGuy** [comment]
Apologies I misunderstood the question. Yes silhouettes and flags are not distributed with the Docker image and at current the Docker image has no way to upload the needed files.
I will look into licensing to see if they can be distributed or if there needs to be a method for each user to individually upload the data.

**Comment by TheBiggerGuy** [comment]
Could you please try the latest Dev build. I have updated VRS to now download and use the route DB. Please set the required lat/long vars in the ```variables-virtualradar.env``` file.

To run against the Dev branch please use: ```TAG=dev docker-compose pull/build/up```

Please note: At current any configuration and/or plane tracking history is lost with restarts/updates. I aim to update this in future to maintain it across updates/restarts.

![Screenshot from 2019-05-02 14-59-21](https://user-images.githubusercontent.com/152316/57080684-efa18d80-6cea-11e9-99e6-28b9a9306afd.png)

Notes:
* https://github.com/TheBiggerGuy/docker-ads-b/commit/b5f3944bc0bfa99f69485ba73d13ea8b176e0d31
* https://travis-ci.org/TheBiggerGuy/docker-ads-b/jobs/527320658

**Comment by TheBiggerGuy** [comment]
Also of note: The route DB will not work immediately. For an unknown reason the downloaded DB is missing tables. After 24 hours the server should auto update the file and get the working results as shown above. I still need to root cause this.

**Comment by TheBiggerGuy** [comment]
This has been working well for me so merging into latest. https://github.com/TheBiggerGuy/docker-ads-b/pull/17
Still need to fix silhouettes and flags.
